tests/testsuite/cargo_report_sessions/mod.rs

```diff
@@ -91,13 +82,10 @@ fn in_workspace() {
 
     p.cargo("report sessions -Zbuild-analysis")
         .masquerade_as_nightly_cargo(&["build-analysis"])
-        .with_status(1)
         .with_stderr_data(str![[r#"
-[ERROR] unrecognized subcommand 'sessions'
-
-Usage: cargo report [OPTIONS] <COMMAND>
+Session IDs for `[ROOT]/foo` (showing up to 10):
```

Comment:
In `cargo search`, we expose the limit as
```
... and 10792 crates more (use --limit N to see more)
```

Reply:
Noted that in the PR description as a design decision to raise the visibility in case other people ask